clusterctl-settings.json

```diff
@@ -2,6 +2,6 @@
   "name": "infrastructure-vsphere",
```

Comment:
@sbueringer i double checked these tasks, looks like those are already done in main branch.

Reply:
Oh thx. I was just checking these files while I had your PR checked out :) Didn't think that it might be already done on main :D 

Thx for checking!